src/Infastructure/Infastructure.csproj

```diff
@@ -19,6 +19,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <ProjectReference Include="..\..\..\cardanosharp-dbsync-efcore\CardanoSharp.DbSync.EFCore.csproj" />
```

Comment:
@Daniel-SchaeferJ Just note this relative path. 
![image](https://user-images.githubusercontent.com/638507/117387083-2603db80-aeb6-11eb-868b-4348a43bcaa3.png)
You can see how my library and demo live along side each other. this will eventually be moot and we'll have a nuget

Reply:
Sounds good! I see how you referenced now and it'll be nice to have it isolated from the project even though I meant to do that haha